plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -159,8 +166,19 @@ export const SearchResult = ({ searchQuery }: SearchResultProps) => {
       setFilteredResults(withFilters);
     }
   }, [filters, searchQuery, results]);
-
-  if (loading || error || !results) return null;
+  if (loading) {
+    return <Progress />;
+  }
+  if (error) {
+    return (
+      <Alert severity="error">
+        Error encountered while fetching search results. {error.toString()}
+      </Alert>
+    );
+  }
+  if (!results || results.length === 0) {
+    return <EmptyState missing="info" title="Sorry, no results were found" />;
```

Comment:
```suggestion
    return <EmptyState missing="data" title="Sorry, no results were found" />;
```

Reply:
so that we get this storybook component https://backstage.io/storybook/?path=/story/feedback-emptystate--data